src/js/index.js

```diff
@@ -1,22 +1,6 @@
 import '../styles/style.scss';
-import Icon from '../assets/icon.png';
-import printMe from './print';
+import Header from './Header/Header'
 
-function component() {
-  const element = document.createElement('div');
-  const btn = document.createElement('button');
-
-  const myIcon = new Image();
-  myIcon.src = Icon;
-
-  element.appendChild(myIcon);
-
-  btn.innerHTML = 'Click me and check the console!';
-  btn.onclick = printMe;
-
-  element.appendChild(btn);
-
-  return element;
-}
-
-document.body.appendChild(component());
+export {
+  Header,
+}
```

Comment:
So here we just attach our Header component to the root element of our application. In the feature our app will grow and we gonna attach more components here.
```suggestion
function init() {
  const rootElement = document.querySelector('#root');
  rootElement.appendChild(Header());
}

init();

```